config/ares-broker.yaml

```diff
@@ -0,0 +1,26 @@
+port: 9474
+
+http:
+  max_connections: 300
+  read_time_out_in_seconds: 20
+  write_time_out_in_seconds: 300 # 5 minutes to write the result
+
+controller:
```

Comment:
is this for muttley, it also internal stuff. for oss, we just need use host:port. The http endpoint creation likely need some wrap

Reply:
yes this is for muttley. but it's using http header which is generic enough for OSS. I can remove the uber specific header from config file though.